modules/special-rules/pages/transport-x.adoc

```diff
@@ -1,2 +1,16 @@
 = Transport (X)
 
+{blank}[TODO: Add introductory description.]
+
+---
+
+Where your unit has the {transport} ability:
+
+* The {transport} unit can carry as many Infantry units as we state in brackets.
+For example, the Rhino has {transport-2} -- so you can embark up to two Infantry units in a Rhino.
+
+NOTE: A {transport} unit does not gain the abilities of embarked units.
```

Comment:
There needs to be some form of notification that this rule is incomplete, and that players **must** also read The Movement phase/Transport. The Related information link is not enough by itself.

Note that hitch-x was written at the same time as transport-x, and will need the equivalent changes that you applied to transport-x.

Reply:
I've added a suggestion above to resolve your first para.

I've made a local draft on Hitch (X) to align with this and the rest of Transport (X) — I'll push it shortly.

So, I'll resolve this conversation.